libraries/app/include/scorum/app/state.hpp

```diff
@@ -153,7 +152,7 @@ FC_REFLECT( scorum::app::account_vote, (authorperm)(weight)(rshares)(percent)(ti
 
 FC_REFLECT( scorum::app::discussion_index, (category)(trending)(payout)(payout_comments)(trending30)(updated)(created)(responses)(active)(votes)(maturing)(best)(hot)(promoted)(cashout) )
 FC_REFLECT( scorum::app::tag_index, (trending) )
-FC_REFLECT_DERIVED( scorum::app::discussion, (scorum::app::comment_api_obj), (url)(root_title)(pending_payout_value)(total_pending_payout_value)(active_votes)(replies)(author_reputation)(promoted)(body_length)(reblogged_by)(first_reblogged_by)(first_reblogged_on) )
+FC_REFLECT_DERIVED( scorum::app::discussion, (scorum::app::comment_api_obj), (url)(root_title)(pending_payout_value)(active_votes)(replies)(author_reputation)(promoted)(body_length)(reblogged_by)(first_reblogged_by)(first_reblogged_on) )
```

Comment:
Try to use column order:

(url)
(root_title)
(pending_payout_value)
...

with '// clang-format off'